lib/api/request.js

```diff
@@ -0,0 +1,12 @@
+class Request {
+
+  createHeaders(accessToken) {
```

Comment:
I prefer `buildHeaders` here. But it's up to you...

Reply:
je to factory metoda, tak to nechám tak jak je. `build` by se hodil pro builder pattern.

Reply:
Wie bitte? :-D

Reply:
Oh shit, it's public! we shouldn't use bad words and Czech! :-O 

I said: "It's a factory method. `build` is usually used in combination with the builder pattern".